dtc-overlay.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh -e
```

Comment:
So this script just installs `dtc`? If so, let's rename it to `dtc-install.sh` or something similar.

Reply:
Updated to dtc_installer.sh